ports/marzbanpp/vcpkg.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "marzbanpp",
+  "version": "1.0.1",
+  "description": "marzbanpp is the C++ library for interaction with Marzban through its REST API.",
+  "homepage": "https://github.com/andrascii/marzbanpp",
+  "license": "MIT",
+  "dependencies": [
+    "curl",
```

curl features subject to user choice:
```suggestion
    {
      "name": "curl",
      "default-features": false
    },
```